src/module/AddRemove.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable no-loop-func */
+let List = [];
+let isEditing = false;
+let todoEdit = null;
+
+const saveData = () => {
+  localStorage.setItem('listitem', JSON.stringify(List));
+};
+
+const retrivedata = () => {
+  const localFormData = JSON.parse(localStorage.getItem('listitem'));
+  if (localFormData == null) {
+    List = [];
+  } else {
+    List = localFormData;
+  }
+};
+
+const editList = (todo) => {
+  isEditing = true;
+  todoEdit = todo;
+  const desc = document.getElementById('addtodo');
+  desc.value = todo.description;
+  desc.focus();
+};
+
+const removeList = (indexID) => {
+  List = List.filter((ind) => ind.index !== indexID);
+  List = List.map((todo, index) => ({
+    completed: todo.completed,
+    description: todo.description,
+    index: index + 1,
+  }));
+  // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Kindly avoid disabling linters because they guide us in writing code that is clean and well formatted.